compose.yaml

```diff
@@ -1,10 +1,27 @@
 services:
   mongodb:
     image: 'mongo:7.0.4-jammy'
+    container_name: flashcards-mongodb
     ports:
       - '27017:27017'
     volumes:
       - 'mongo-data:/data/db'
+    networks:
+      - flashcards-network
+  authserver:
+    image: 'wisskirchenj/flashcards:0.0.1-SNAPSHOT'
```

Comment:
Does this image includes the complete application or the auth service only? If the latter, it might be a good idea to set a more descriptive name.

Reply:
true  - also saw the point myself  - but wanted to get things running  -and native build and new push takes time...
I' ll fix this now I guess.

Reply:
fixed it - image  now named flashcardsauth-server:0.0.1-SNAPSHOT.
BTW - the idea to put the flashcards app itself in container too is not too bad - maybe for another issue :-)